frontend/src/components/modals/NewSchemaDialog.vue

```diff
@@ -97,7 +97,10 @@ export default {
         this.schemaUrl = "";
         this.closeModal();
       } catch (error) {
-        this.error = error;
+        this.error = {
+          message:
+            "unable to add schema url (ensure the url is a valid schema url",
```

Comment:
Do we have any standard approach to message formatting? Might be better to start sentence with capital